README.md

```diff
@@ -4,6 +4,62 @@ Ingests a static GTFS feed and outputs useful geoJSON files. It supports excludi
 
 #### Output Files
 * `transit-service-area.json`: A polygon that defines the area within which transit data is known.
+* `route-line-lookup.json`: Dictionaries used at runtime by `bikehopper-web-app`. 
+    1. routeTripShapeLookup: 
+        This is a two-level dictionary
+        Level1 :
+            Key is a route-id, Value is the 2nd Level dictionary
+        Level 2:
+            Key is a trip-id, Value is a shape-id
+    2. shapeIdLineStringLookup:
+        Key is the shape-id of a route, and the value is a LineString of the entire route
+    3. tripIdStopIdsLookup:
+        Key is a trip-id, and value is an array of stop-ids for that trip
+* `/route-tiles`: A `{z}/{x}/{y}` VectorTile tileset that contains LineStrings for all the routes in the GTFS feed
+```
+"vector_layers": [
+    {
+        "id": "route-lines",
+        "description": "",
+        "minzoom": 7,
+        "maxzoom": 14,
+        "fields": {
+            "route_color": "String",
+            "route_id": "String",
+            "route_text_color": "String",
+            "trip_ids": "String"
+        }
+    }
+],
+```
+* `/stop-tiles`: A `{z}/{x}/{y}` VectorTile tileset that contains the Points of all the Stops in the GTFS feed
+```
+"vector_layers": [
+    {
+        "id": "stops",
+        "description": "",
+        "minzoom": 8,
+        "maxzoom": 14,
+        "fields": {
+            // Encoding the types of modalities at the stop
+            // Keys are taken from https://gtfs.org/documentation/schedule/reference/#routestxt
```

Comment:
I'd still prefer you used the numbers. That way, it's clearer that GTFS is the source of truth, which makes us go to the GTFS spec to see what the numbers mean, which will make us remember that, for example, we have to account for trolleybus as well as bus.

I'll accept this citation as enough of a compromise not to block on this, but it's still something that could be overlooked at the site of consumption of the data.

Maybe in the future when we combine gtfs-processor, bikehopper-web-app and bikehopper-ui into one repo, we can have a common type specification for this to mitigate the issue.